docs/extensions-plugins/obsidian.mdx

```diff
@@ -103,51 +103,50 @@ export const HeaderLogoImage = (props) => {
 
 _Pieces For Developers captures **NO identifiable user data**. Our **Local-only** architecture means your notes never have to leave your device._
 
-_The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Suite</a> must be downloaded and installed separately from the Obsidian Plugin in order for the Plugin to work properly._
-
 ## Install Instructions
 
-[Installation Instructions](https://github.com/pieces-app/obsidian-pieces#install-instructions) for the latest Pieces for Obsidian
-release:
+Installation instructions for the latest Pieces for Developers Obsidian Plugin release:
 
-1. Make sure you have community plugins enabled in Obsidian.
-2. Search for **Pieces for Developers** in the Obsidian community plugin store.
-3. Click `install` on the Pieces for Developers plugin listing.
-4. After the plugin is downloaded and installed, click `enable` to start the plugin.
-5. Now you can get started with the plugin! Just click the Pieces icon on the left, and start the magic!
+1. Make sure you have Pieces OS installed
```

Comment:
Is it possible to add a hyperlink here to the appropriate page in the docs for them to get Pieces OS?

Reply:
The best link I see is this: https://docs.pieces.app/~161/installation-getting-started/what-am-i-installing#pieces-desktop-app--pieces-os

Reply:
That's perfect